[2024-06-03T21:53:10.685+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T21:53:10.703+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [queued]>
[2024-06-03T21:53:10.709+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [queued]>
[2024-06-03T21:53:10.710+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T21:53:10.721+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-03-01 00:00:00+00:00
[2024-06-03T21:53:10.728+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=26835) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T21:53:10.730+0000] {standard_task_runner.py:63} INFO - Started process 26837 to run task
[2024-06-03T21:53:10.730+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-03-01T00:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpkzjpr8je']
[2024-06-03T21:53:10.731+0000] {standard_task_runner.py:91} INFO - Job 117: Subtask blunders_etl
[2024-06-03T21:53:10.778+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T21:53:10.854+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-01T00:00:00+00:00'
[2024-06-03T21:53:10.855+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T21:53:10.857+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T21:53:10.865+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:06:50.643+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:06:50.644+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:06:50.653+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-03-01T00:00:00+00:00, execution_date=20210301T000000, start_date=20240603T215310, end_date=20240603T220650
[2024-06-03T22:06:50.681+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:06:50.693+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:06:50.694+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:12:41.018+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:12:41.033+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [queued]>
[2024-06-08T19:12:41.039+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [queued]>
[2024-06-08T19:12:41.040+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:12:41.051+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-03-01 00:00:00+00:00
[2024-06-08T19:12:41.058+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=26457) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:12:41.060+0000] {standard_task_runner.py:63} INFO - Started process 26459 to run task
[2024-06-08T19:12:41.059+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-03-01T00:00:00+00:00', '--job-id', '293', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp2etw0pyf']
[2024-06-08T19:12:41.060+0000] {standard_task_runner.py:91} INFO - Job 293: Subtask blunders_etl
[2024-06-08T19:12:41.093+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:12:41.157+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-01T00:00:00+00:00'
[2024-06-08T19:12:41.158+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:12:41.159+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:12:41.167+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T19:26:06.196+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T19:26:06.197+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T19:26:06.208+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-03-01T00:00:00+00:00, execution_date=20210301T000000, start_date=20240608T191241, end_date=20240608T192606
[2024-06-08T19:26:06.258+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T19:26:06.270+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T19:26:06.272+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:18:15.312+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:18:15.326+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [queued]>
[2024-07-08T01:18:15.332+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [queued]>
[2024-07-08T01:18:15.333+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:18:15.342+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-03-01 00:00:00+00:00
[2024-07-08T01:18:15.349+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=28051) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:18:15.350+0000] {standard_task_runner.py:63} INFO - Started process 28053 to run task
[2024-07-08T01:18:15.350+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-03-01T00:00:00+00:00', '--job-id', '661', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp7vd6usyy']
[2024-07-08T01:18:15.351+0000] {standard_task_runner.py:91} INFO - Job 661: Subtask blunders_etl
[2024-07-08T01:18:15.385+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-03-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:18:15.455+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-01T00:00:00+00:00'
[2024-07-08T01:18:15.456+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:18:15.457+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:18:15.465+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T01:31:02.310+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T01:31:02.310+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T01:31:02.318+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-03-01T00:00:00+00:00, execution_date=20210301T000000, start_date=20240708T011815, end_date=20240708T013102
[2024-07-08T01:31:02.371+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T01:31:02.391+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T01:31:02.392+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
